### ISSUE TITLE
Cannot init new project or existing project from CLI

### ISSUE THREAD

**Issue by wontwon** [graphcool/graphcool#581]
**What is the current behavior?**
 *  After installing graphcool-cli @beta, running graphcool init for a new project returns
```    Creating project Streamvenom Knave... ✔
 ▸    unacceptable kind of an object to dump [object Undefined]
```

* Also, when running graphcool init on an existing project also returns the same as above

**Please share the relevant part of your project (GraphQL schema, functions, permissions, ...) for easier reproduction**
 * Currently running: 
```  graphcool/1.3.16-beta.20 (darwin-x64) node-v8.3.0  ```

**What is the expected behavior?**
 * After running graphcool init, I should be able to start a brand new project.

**Comment by marktani** [comment]
can confirm:

![image](https://user-images.githubusercontent.com/1780597/30785072-bf0978ec-a160-11e7-8ce3-1078711b1447.png)

**Comment by timsuchanek** [comment]
Thanks a lot for reporting this @wontwon and also thanks @marktani!
I just released a new version of the CLI (`graphcool@1.3.16-beta.21`) where this bug is fixed.

```
npm install -g graphcool@beta
```
